AccountingSystem/private/DBUtils.py

```diff
@@ -31,11 +30,13 @@ def _retrieveBucketedData( self,
                        ( "%s, %s", ( "field1name", "field2name", "field3name" )
     """
     validCondDict = {}
-    if type( condDict ) == types.DictType:
+    if isinstance(condDict, dict):
       for key in condDict:
-        if type( condDict[ key ] ) in ( types.ListType, types.TupleType ) and len( condDict[ key ] ) > 0:
+        if isinstance(condDict[ key ], (list, tuple)) and len(condDict[key]):
```

Comment:
Can ```len(condDict[key])``` be replaced by just ```condDict[key]``` ?

Reply:
Probably yes, for this specific case. I will change it.